semanticdb/metap/src/main/scala/scala/meta/internal/metap/SyntheticPrinter.scala

```diff
@@ -1,28 +1,84 @@
 package scala.meta.internal.metap
 
-import scala.math.Ordering
-import scala.meta.internal.semanticdb._
+import scala.meta.internal.{semanticdb => s}
 
-trait SyntheticPrinter extends BasePrinter with OccurrencePrinter {
-  def pprint(synth: Synthetic): Unit = {
+trait SyntheticPrinter
+    extends BasePrinter
+    with RangePrinter
+    with SymbolInformationPrinter
+    with ConstantPrinter {
+  self =>
+
+  private def shorten(s: String): String = {
+    val newline = s.indexOf('\n')
+    if (newline == -1) {
+      if (s.length > 100) s.takeRight(100) + "..."
+      else s
+    } else s.substring(0, newline) + "..."
+  }
+
+  def pprint(synth: s.Synthetic): Unit = {
     opt(synth.range)(pprint)
-    opt(": ", doc.substring(synth.range))(out.print)
+    opt(": orig(", doc.substring(synth.range), ")")(out.print)
```

Comment:
Let's drop the `orig` thing here, since there's no `OriginalTree` involved.

Reply:
I like the symmetry it has on the printouts, and it is still a quote from the original source.